### DISCUSSION TITLE
Solver Recomendation for CLC

### DISCUSSION THREAD

**Discussion by alguzur** [IDAES/idaes-pse#422] [Q&A] [answered]
Hi Everyone!,
I am currently trying to optimize a flowsheet of two BFB reactors interconnected forming a loop (CLC). However, it has been very challenging to get a global optimum solution, all that I am getting is locally feasible solutions that I am using as the next initialization values one after another. Then I would like to know if I can adapt a different solver for non-linear optimization that can handle this CLC flowsheet in a more efficient way?. Or there is a commercial solver that I could purchase for this issue?. Thanks a lot!

**Comment by andrewlee94** [comment, marked as answer]
The first question here is what solver are you currently using to solve the model? If you are using IPOPT (our default solver), then this is not a global solver and you will need to use something different. The `ideas get-extensions` command get you access to compiled versions of all the open-source solvers from COIN-OR, so there are alternatives you could try (e.g. Couenne) however I am not an expert on the strengths of each solver.

AS a suggestion, I would still use IPOPT for the initialization (as this is what we use for testing so it is most likely to work), and then switch to a global solver for the optimisation.

**Reply by alguzur** [reply]
Yes Andrew, I am using IPOPT currently. Thanks for the suggestion, then I will try to run the flowsheet again following the routine you mention: Initialization IPOPT and then optimization via Counne or others.
